pkg/cmdimpl/add_test.go

```diff
@@ -192,15 +192,70 @@ var _ = Describe("Test helm manifest", func() {
 
 		fluxops.SetFluxHandler(fakeHandler)
 
-		params.Name = "simple-name"
 		params.Name = "simple-name"
 		params.Path = "./my-chart"
 		params.Namespace = "wego-system"
 
-		Expect(generateHelmManifest()).Should(Equal([]byte("foo")))
+		Expect(generateHelmManifestGit()).Should(Equal([]byte("foo")))
 	})
 })
 
+var _ = Describe("Test helm manifest from helm repo", func() {
+	It("Verify helm manifest generation from helm ", func() {
+
+		expected := `create helmrelease simple-name \
+			--source="HelmRepository/simple-name" \
+			--chart="testchart" \
+			--interval=5m \
+			--export \
+			--namespace=wego-system`
+
+		fakeHandler := &fluxopsfakes.FakeFluxHandler{
+			HandleStub: func(args string) ([]byte, error) {
+				Expect(args).Should(Equal(expected))
+				return []byte("foo"), nil
+			},
+		}
+
+		fluxops.SetFluxHandler(fakeHandler)
+
+		params.Name = "simple-name"
+		params.Namespace = "wego-system"
+		params.Chart = "testchart"
+
+		Expect(generateHelmManifestHelm()).Should(Equal([]byte("foo")))
+	})
+
+})
+
+var _ = Describe("Test helm source from helm repo", func() {
+	It("Verify helm source generation from helm ", func() {
+
+		expected := `create source helm test \
+			--url="https://github.io/testrepo" \
+			--interval=30s \
+			--export \
+			--namespace=wego-system `
+
+		fakeHandler := &fluxopsfakes.FakeFluxHandler{
+			HandleStub: func(args string) ([]byte, error) {
+				Expect(args).Should(Equal(expected))
+				return []byte("foo"), nil
+			},
+		}
+
+		fluxops.SetFluxHandler(fakeHandler)
```

Comment:
I made a change in the other helm test in my stuff to not set the fake handler directly (using `WithOverrides` instead) because it leaves the flux handler broken for other tests.